test/nl/hannahsten/texifyidea/inspections/bibtex/BibtexUnusedEntryInspectionTest.kt

```diff
@@ -0,0 +1,27 @@
+package nl.hannahsten.texifyidea.inspections.bibtex
+
+import nl.hannahsten.texifyidea.inspections.TexifyInspectionTestBase
+import nl.hannahsten.texifyidea.testutils.writeCommand
+
+class BibtexUnusedEntryInspectionTest : TexifyInspectionTestBase(BibtexUnusedEntryInspection()) {
+
+    override fun getTestDataPath(): String {
+        return "test/resources/inspections/bibtex/unusedbibentry"
+    }
+
+    fun `test warnings where needed`() {
+        myFixture.configureByFiles( "references.bib", "main.tex")
```

Comment:
```suggestion
        myFixture.configureByFiles("references.bib", "main.tex")
```
the linter should have found this

Reply:
It did. And I was fixing it, but then my laptop died while I was typing the commit message. Oops.